src/main/webapp/resources/js/apis/analysis/analysis.js

```diff
@@ -236,6 +236,19 @@ export function downloadOutputFile({ submissionId, fileId }) {
   );
 }
 
+/**
+ * Get the newick string for the submission.
```

Comment:
Add a `@param` comment

Reply:
Added in cba6f9588a746bd177bea92e49a052caa6586fe5